src/shared/constants.h

```diff
@@ -7,75 +7,52 @@
-const double ROBOT_MAX_ACCELERATION_METERS_PER_SECOND_SQUARED = 3.0;
-// The maximum angular acceleration achievable by our robots, in radians per second
-// squared
-const double ROBOT_MAX_ANG_ACCELERATION_RAD_PER_SECOND_SQUARED = 10.0;
+static const unsigned MAX_ROBOTS_OVER_RADIO = 8;
 
 // The maximum speed attainable by enemy robots
-const double ENEMY_ROBOT_MAX_SPEED_METERS_PER_SECOND = 3.0;
+static const double ENEMY_ROBOT_MAX_SPEED_METERS_PER_SECOND = 3.0;
 // The maximum acceleration achievable by enemy robots, in metres per seconds squared.
-const double ENEMY_ROBOT_MAX_ACCELERATION_METERS_PER_SECOND_SQUARED = 4.0;
+static const double ENEMY_ROBOT_MAX_ACCELERATION_METERS_PER_SECOND_SQUARED = 4.0;
 
-const double ACCELERATION_DUE_TO_GRAVITY_METERS_PER_SECOND_SQUARED = 9.81;
+static const double ACCELERATION_DUE_TO_GRAVITY_METERS_PER_SECOND_SQUARED = 9.81;
 
 /* Unit Conversion */
-const double MILLIMETERS_PER_METER = 1000.0;
-const double METERS_PER_MILLIMETER = 1.0 / 1000.0;
-const double CENTIMETERS_PER_METER = 100.0;
-const double METERS_PER_CENTIMETER = 1.0 / 100.0;
-
-const double CENTIRADIANS_PER_RADIAN = 100.0;
-const double RADIANS_PER_CENTIRADIAN = 1.0 / 100.0;
-
-const double NANOSECONDS_PER_MILLISECOND  = 1000000.0;
-const double NANOSECONDS_PER_SECOND       = 1000000000.0;
-const double MICROSECONDS_PER_MILLISECOND = 1000.0;
-const double MICROSECONDS_PER_SECOND      = 1000000.0;
-const double MILLISECONDS_PER_SECOND      = 1000.0;
-const double SECONDS_PER_MICROSECOND      = 1.0 / 1000000.0;
-const double SECONDS_PER_MILLISECOND      = 1.0 / 1000.0;
-const double MILLISECONDS_PER_MICROSECOND = 1.0 / 1000.0;
-
-// Indefinite dribbler mode sets a speed that can be maintained indefinitely
-const double INDEFINITE_DRIBBLER_SPEED = 1000.0;
-// Max force dribbler mode sets the speed that applies the maximum amount of force on the
-// ball
-const double MAX_FORCE_DRIBBLER_SPEED = 16000.0;
-
-const double POSSESSION_TIMESTAMP_TOLERANCE_IN_MILLISECONDS = 10;
+static const double MILLIMETERS_PER_METER = 1000.0;
+static const double METERS_PER_MILLIMETER = 1.0 / 1000.0;
+static const double CENTIMETERS_PER_METER = 100.0;
+static const double METERS_PER_CENTIMETER = 1.0 / 100.0;
+
+static const double CENTIRADIANS_PER_RADIAN = 100.0;
+static const double RADIANS_PER_CENTIRADIAN = 1.0 / 100.0;
+
+static const double NANOSECONDS_PER_MILLISECOND  = 1000000.0;
+static const double NANOSECONDS_PER_SECOND       = 1000000000.0;
+static const double MICROSECONDS_PER_MILLISECOND = 1000.0;
+static const double MICROSECONDS_PER_SECOND      = 1000000.0;
+static const double MILLISECONDS_PER_SECOND      = 1000.0;
+static const double SECONDS_PER_MICROSECOND      = 1.0 / 1000000.0;
+static const double SECONDS_PER_MILLISECOND      = 1.0 / 1000.0;
+static const double MILLISECONDS_PER_MICROSECOND = 1.0 / 1000.0;
+
+static const double POSSESSION_TIMESTAMP_TOLERANCE_IN_MILLISECONDS = 10;
```

Comment:
just a thought, we could remove `_IN_` from this variable name. Out-of-scope, your call

Reply:
this constant isn't even used so removed